actions/reviews/CheckClassReadme.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2022 Oracle and/or its affiliates.
+# All rights reserved. The Universal Permissive License (UPL), Version 1.0 as shown at http://oss.oracle.com/licenses/upl
+# CheckClassReadme.py
+# Description: Includes the logic to check if Readme has been updated with corresponding class during a PR
+
+from actions.utils.helpers.helper import *
+import json
+import os
+
+def check_readme():
+
+    files_per_class = get_class_files()
+
+    for key in files_per_class:
+        class_files = files_per_class[key]
+        readme_path = get_readme_path(key)
+        readme_items = []
+        if os.path.exists(readme_path):
+            with open(readme_path) as f:
+                for line in f:
+                    if '.py' in line:
+                        name = line.split('.py')[0].split('[')[1]
+                        readme_items.append(name)
```

Comment:
This part correponds to the analysis of this Check. Can we decouple this on a inner function named analyze():